package.json

```diff
@@ -47,6 +47,7 @@
   "dependencies": {
     "@google-cloud/projectify": "^0.3.2",
     "@google-cloud/promisify": "^0.4.0",
+    "@types/duplexify": "^3.6.0",
```

Comment:
should this be a dev dependency?

Reply:
*sighs* no.  That's the error this uncovered.  `gax` is returning this somewhere in a class that is being *extended* here.

Reply:
That sounds .. weird? Sorry if I'm misunderstanding, buuut would it better if we just made a change that casts said stream as a plain old Duplex?

Reply:
It's possible that's the right change *upstream* in `gax`.  The real problem here is that gax leaks types from other packages.  The "correct" fix is probably to make this change in *gax* instead of here. 

*scratches head*

Thoughts?

Reply:
SGTM!